schemas/Manual.regions.schema.json

```diff
@@ -39,6 +39,32 @@
                             "type":"boolean",
                             "default": false
                         },
+                        "exit_requires": {
+                            "description": "(Optional) Additional requirements to use a specific exit from this region.",
```

Comment:
We should make these descriptions more plain and have them incorporate language that links them to `connects_to`, since they're gating that setting. Something like "to exit this region to a connected region."

And for the entrance_requires, something like "to enter this region from a region that connects to this one." Tweak as needed.